plugins/modules/monitoring/pagerduty_change.py

```diff
@@ -0,0 +1,173 @@
+    required: true
+    type: str
+  summary:
+    description:
+      - A short description of the change that occurred.
+    required: true
+    type: str
+  source:
+    description:
+      - The source of the change event.
+    required: false
+    default: Ansible
+    type: str
+  user:
+    description:
+      - The name of the user or process that triggered this deployment.
+    required: false
+    type: str
+  repo:
+    description:
+      - The URL of the project repository.
+    required: false
+    type: str
+  revision:
+    description:
+      - An identifier of the revision being deployed, typically a number or SHA from a version control system.
+    required: false
+    type: str
+  environment:
+    description:
+      - The environment name, typically 'production', 'staging', etc.
+    required: false
+    type: str
+  link_url:
+    description:
+      - A URL where more information about the deployment can be obtained.
+    required: false
+    type: str
+  link_text:
+    description:
+      - Descriptive text for a URL where more information about the deployment can be obtained.
+    required: false
+    type: str
+  url:
+    description:
+      - URL to submit the change event to.
+    required: false
+    default: https://events.pagerduty.com/v2/change/enqueue
+    type: str
+  validate_certs:
+    description:
+      - If C(no), SSL certificates for the target URL will not be validated.
+        This should only be used on personally controlled sites using self-signed certificates.
+    required: false
+    default: yes
+    type: bool
+'''
+
+EXAMPLES = '''
+- name: Track the deployment as a PagerDuty change event
+  community.general.pagerduty_change:
+    integration_key: abc123abc123abc123abc123abc123ab
+    summary: The application was deployed
+    user: ansible
+    revision: '4.2'
+'''
+
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils.basic import AnsibleModule
+from datetime import datetime
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            integration_key=dict(required=True, type='str'),
+            summary=dict(required=True, type='str'),
+            source=dict(required=False, default='Ansible', type='str'),
+            user=dict(required=False, type='str'),
+            repo=dict(required=False, type='str'),
+            revision=dict(required=False, type='str'),
+            environment=dict(required=False, type='str'),
+            link_url=dict(required=False, type='str'),
+            link_text=dict(required=False, type='str'),
+            url=dict(required=False,
+                     default='https://events.pagerduty.com/v2/change/enqueue', type='str'),
+            validate_certs=dict(default=True, type='bool')
+        ),
+        supports_check_mode=True
+    )
+
+    if module.check_mode:
+        module.exit_json(changed=True)
```

Comment:
```suggestion
    if module.check_mode:
        module.exit_json(changed=True)
```
not sure this is fair. maybe we should check anyhow, at least, the API's availability?